console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -66,14 +66,32 @@ class UserController {
     return (this.delegation.orgs.length !== 0) && (this.delegation.roles.length !== 0)
   }
 
+  // search each choosen span elements and set title manually
+  // to display roles description on hover
+  setTitle () {
+    if (this.systemRoles) {
+      let selectEls = $('li.search-choice')
+      $(selectEls).each(i => {
```

Comment:
no need for `$` it's already a jquery object.
`.search-choice span` selector would avoid to fetch first child.